tests/test_utils/test_schemaview.py

```diff
@@ -1816,17 +1962,71 @@ def test_dynamic_enum(schema_view_with_imports: SchemaView) -> None:
     assert set(e.include[0].reachable_from.source_nodes) == {"GO:0007049", "GO:0022403"}
 
 
-def test_get_elements_applicable_by_identifier(schema_view_no_imports: SchemaView) -> None:
+# dictionary mapping class name to id_prefixes
+KS_PREFIXES_BY_CLASS = {PERSON: {"orcid", "doi", "zfin", "wb"}, ORGANIZATION: {"pmid", "zfin", "wb"}}
```

Comment:
These values must remain aligned with those defined in the input file `kitchen_sink_noimports.yaml`, right? Wouldn't it be meaningful to generate this dictionary with PyYAML out of the schema? That way the expectation is always up-to-date with the input.